xml/net_dhcp.xml

```diff
@@ -709,7 +709,7 @@ fixed-address &wsIip;;
     </listitem>
     <listitem>
      <para>
-      <filename>/etc/resolv.conf</filename>
+      <filename>/var/run/netconfig/resolv.conf</filename>
```

Comment:
This one definitely is /etc/resolv.conf, because that's what the startup-script /usr/lib/dhcp/dhcp actually copies over to the chroot. At least in dhcp-server-4.3.5-4.15, maybe the package should be adjusted?! I don't know and I don't think so because the canonical location /etc/resolv.conf works fine and will continue to.